docs/BR.md

```diff
@@ -1403,23 +1403,23 @@ The business continuity plan MUST include:
 ### 6.1.1 Key pair generation
 
 #### 6.1.1.1 CA Key Pair Generation
-For Root CA Key Pairs that are either (i) used as Root CA Key Pairs or (ii) Key Pairs generated for a subordinate CA that is not the operator of the Root CA or an Affiliate of the Root CA, the CA SHALL:
+For CA Key Pairs that are either (i) used as a CA Key Pair for a Root Certificate or (ii) used as a CA Key Pair for a Subordinate CA Certificate, where the Subordinate CA is not the operator of the Root CA or an Affiliate of the Root CA, the CA SHALL:
 
 1. prepare and follow a Key Generation Script,
-2. have a Qualified Auditor witness the Root CA Key Pair generation process or record a video of the entire Root CA Key Pair generation process, and
+2. have a Qualified Auditor witness the CA Key Pair generation process or record a video of the entire CA Key Pair generation process, and
 3. have a Qualified Auditor issue a report opining that the CA followed its key ceremony during its Key and Certificate generation process and the controls used to ensure the integrity and confidentiality of the Key Pair.
```

Comment:
Typically a technically-constrained CA can be internally audited by the CA and is not in direct scope of independent audits. I think it should be considered an exception in this case for the need to have the qualified auditor's report as a particular activity, as in any case the creation of any subordinate will be already reviewed during the period audits and that should be enough.

Reply:
That's an very fair point!

Given that it would be explicitly more permissive than the current requirements, rather than more restrictive, it might be worth a separate ballot (than cleanups and clarifications), since it's not really either at that point. That is, I'm not aware of any root programs treating this as exempting TCSCs and bearing clarification of expectations, but I could be wrong.

However, there's also another, trickier issue to solve. Trying to word an exception for TCSCs creates a risk that folks will misinterpret such an exception, and see the following sequence as valid:

1. Widget Corp generate a CA Key Pair (without any of these requirements, effectively)
2. Sprockets Inc, the Root CA, issues a TCSC to Widget Corp (permitted by the proposed exemption)
3. Widget Corp then later applies for an unconstrained Sub CA / unconstrained Root, and references this section to argue that they complied with the requirements "at time of key generation", and since their new unconstrained CA is not generating new keys, they're exempt from these requirements.

That's obviously not the intent, but a reading I worry CAs would make, especially given the trouble we've seen with interpreting requirements for test websites.

To make sure we don't lose this discussion, I filed https://github.com/cabforum/documents/issues/187 with some thoughts, but left it a bit open-ended. We should probably discuss further on the list, since getting this right could be tricky.